config/config.md

```diff
@@ -12,7 +12,6 @@
 
 | Key | Type | Default | Descriptions |
 | --- | -----| ------- | ----------- |
-| `mode` | String | `standalone` | The running mode of the datanode. It can be `standalone` or `distributed`. |
```

Comment:
Is old configuration file able to start with new greptimedb?

Reply:
Let me try it

Reply:
It seems that standalone can start normally, but the mode configuration item has no effect.